src/main/java/com/github/ilim/backend/ilimEmailDeliveryServ/controller/EmailController.java

```diff
@@ -0,0 +1,35 @@
+package com.github.ilim.backend.ilimEmailDeliveryServ.controller;
+
+
+
+import com.github.ilim.backend.ilimEmailDeliveryServ.model.EmailRequest;
+import com.github.ilim.backend.ilimEmailDeliveryServ.model.EmailResponse;
+import com.github.ilim.backend.ilimEmailDeliveryServ.service.EmailService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+/**
+ * Controller for handling email-related operations.
+ *
+ * Author: praveendran
+ */
+@RestController
+@RequestMapping("/api/email")
+public class EmailController {
+
+    private final EmailService emailService;
+
+    @Autowired
+    public EmailController(EmailService emailService) {
+        this.emailService = emailService;
+    }
+
+    @PostMapping("/send")
+    public ResponseEntity<EmailResponse> sendEmail(@RequestBody EmailRequest request) {
```

Comment:
You implemented a good validation rules for `EmailRequest`, good job.
But they are disabled currently.
To enable them add `@Valid` before `@RequestBody`.

So it becomes like this:
```
public ResponseEntity<EmailResponse> sendEmail(@Valid @RequestBody EmailRequest request) {

```

Reply:
Thanks for pointing out, have addressed the same